workflows/s3_object_lock/SPECIFICATION.md

```diff
@@ -0,0 +1,478 @@
+# Amazon S3 Object Lock Workflow - Technical specification
+
+This document contains the technical specifications for _Amazon S3 Object Lock Workflow_,
+a workflow scenario that showcases AWS services and SDKs. It is primarily intended for the AWS code
+examples team to use while developing this example in additional languages.
+
+This document explains the following:
+
+- Architecture and features of the example workflow.
+- Metadata information for the scenario.
+- Sample reference output.
+
+For an introduction, see the [README.md](README.md).
+
+---
+
+### Table of contents
+
+- [Resources and User Input](#resources-and-user-input)
+- [User input](#user-input)
+- [Output](#output)
```

Comment:
These bookmarks don't seem to exist anymore.

Reply:
Removed.